test/test_helper.rb

```diff
@@ -11,10 +11,3 @@ class ActiveSupport::TestCase
 
   # Add more helper methods to be used by all tests here...
 end
-
-Shoulda::Matchers.configure do |config|
```

Comment:
Why you remove this code?

Reply:
chu4 3 hours ago Member
this whole test directory should be removed we use rspec